src/ontology/mondo-edit.obo

```diff
@@ -348459,13 +348459,15 @@ synonym: "mental retardation, autosomal dominant 31" RELATED DEPRECATED [MONDO:L
 synonym: "mental retardation, autosomal dominant type 31" EXACT DEPRECATED [MONDORULE:2]
 synonym: "MRD31" EXACT ABBREVIATION [DOID:0070061, MONDO:Lexical]
 synonym: "neurodevelopmental disorder with neonatal respiratory insufficiency, hypotonia, and feeding difficulties" EXACT [OMIM:616158]
+synonym: "PURA syndrome" EXACT [https://orcid.org/0000-0001-9310-0163, PMID:28448108]
```

Comment:
@MeeSiing can you confirm that "PURA syndrome" indeed refers only to the "point mutation" version of the disease?